Composer/packages/extensions/obiformeditor/src/Form/widgets/LgEditorWidget.tsx

```diff
@@ -74,11 +74,20 @@ export const LgEditorWidget: React.FC<LgEditorWidgetProps> = props => {
     },
   };
 
-  const diagnostic = lgFile && filterTemplateDiagnostics(lgFile.diagnostics, template)[0];
+  const combineMessage = (diagnostics: Diagnostic[]): string => {
```

Comment:
Is the `combineMessage` here the same as the one inside bfc/indexer? If so we can use the one in bfc/indexer, if not we should rename it.

Reply:
Another thing I'm thinking about is should those dialognostics logic belong to LG editor lib? The LGEditor could just receive the `diagnostics` as an input prop.
You know that the whole FormEditor are under refactoring by Andy, if we insert so many diagnostics logic inside form/LgEditorWidget, they are hard to be merged.

Reply:
Yeah, so I guess everybody likes diagnostic more than errorMsg, then I should refactor this.